build.gradle

```diff
@@ -2,14 +2,14 @@ buildscript {
     ext.kotlinVersion = "1.6.21"
     ext.compileVersion = 31
     ext.buildToolsVersion = "30.0.3"
-    ext.minVersion = 14
+    ext.minVersion = 21
```

Comment:
This was needed due to multidex support + mockk-android needing android 21+. I still need to see if we can split the module min version for different buildTypes.